page/index_cors.html

```diff
@@ -2,7 +2,7 @@
 <html>
   <head>
     <meta charset="utf8">
-    <script src="http://localhost:3032/page/static/pipe/bootstrap.js"></script>
+    <script src="static/pipe/bootstrap.js"></script>
```

Comment:
I think this was a part of a setup @alexmojaki had for testing cors across multiple sites, so unless you need this page I'd just leave it unchanged. [we'll need to write actual tests sometime soon :-)]